pyansys_sphinx_theme/static/css/theme.css

```diff
@@ -179,3 +179,84 @@ p.rubric {
     margin-top: 0.75em;
     margin-bottom: 0.75em;
 }
+
+
+/*
+#############
+*/
+table {
```

Comment:
It looks like all dataframe tables have a `dataframe` class, so you should be able to do something like `table.dataframe {`, and have it just select dataframe tables.